ietf/stats/utils.py

```diff
@@ -230,7 +230,10 @@ def get_meeting_registration_data(meeting):
     """
     num_created = 0
     num_processed = 0
-    response = requests.get(settings.STATS_REGISTRATION_ATTENDEES_JSON_URL.format(number=meeting.number))
+    response = requests.get(
+        settings.STATS_REGISTRATION_ATTENDEES_JSON_URL.format(number=meeting.number),
+        timeout=settings.DEFAULT_REQUESTS_TIMEOUT,
+    )
```

Comment:
The addition of the timeout opens the possibility of raising a `requests.exceptions.Timeout` exception. While this code is currently only called from a `manage.py` task, it might be used elsewhere at a later point.  Please handle the exception.